src/icon/internal.tsx

```diff
@@ -103,12 +103,32 @@ const InternalIcon = ({
 
   const validIcon = name && Object.prototype.hasOwnProperty.call(icons, name);
 
+  function iconMap(name: IconProps.Name) {
+    const genAiFilledIcon = `<svg
```

Comment:
Why can this not be in a normal file like the other icons? Would be good to keep the process consistent.

It also means that this icon does not get passed through our `generate-icons` build task, which (among other things) applies some optimizations on the SVGs.

Reply:
Looks like @just-boris suggested this in an earlier review. Are you sure this was a good idea?

Reply:
It started out as a normal file so that it would be part of the build task, that was my preferred path as well. But Boris pointed out the challenges with that approach along with that fact that we already exclude some svgs from this build task, so it seemed low risk here.

Reply:
But we've never used it for SVGs that are served via the `Icon` component, which is a little different I think.

If it's a one-time thing I'm fine with this approach. If this is going to happen more in the future, I think we need a proper mechanism for it.